FAQ.md

```diff
@@ -0,0 +1,24 @@
+## Frequently Asked Questions
+
+- [Should I add an Xcode project to a gitignore file?](#should-i-add-an-xcode-project-to-a-gitignore-file)
+- [What happens when I switch a branch?](#what-happens-when-i-switch-a-branch)
+
+### Should I add an Xcode project to a gitignore file?
+
+Yes, you should. Adding an Xcode project to a `.gitignore` will eliminate the merge conflicts. If you use a Continuous Integration service, add a `tuist generate` command as a step in your build process.
+
+Note that you can still check in an Xcode project as a middle step.
+
+> When you bootstrap a project via `tuist init` command, Tuist will add an Xcode project to `.gitignore` for you.
+
+### What happens when I switch a branch?
```

Comment:
This is a good one that we can have in the documentation website. I think we can make a distinction here between the website FAQ and the documentation FAQ. The first one is more oriented towards the people that are considering adopting the tool and might have more product-related questions. The ones in the documentation website are more oriented towards those people that are already using the tool.